docs/http-requests.md

```diff
@@ -75,7 +81,7 @@
     `/api/application`
 
     Payload contains `postingId` (e.g. `1`), `userId` (e.g. `1`), `resume` (some
-    hex representation of a file) and `coverLetter` (another hex string string
+    base64 representation of a file) and `coverLetter` (another base64 string string
```

Comment:
@lipovac this is relevant to you. Encode resumes and cover letters in base64, not in hex.